Directory.Build.props

```diff
@@ -18,4 +18,12 @@
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
+ <PropertyGroup>
+  <RepositoryUrl>https://github.com/ionide/FsAutoComplete</RepositoryUrl>
+  <FsDocsLicenseLink>https://github.com/ionide/FsAutoComplete/blob/main/LICENSE.md</FsDocsLicenseLink>
+  <FsDocsReleaseNotesLink>https://github.com/ionide/FsAutoComplete/blob/main/CHANGELOG.md</FsDocsReleaseNotesLink>
+  <PackageProjectUrl>https://ionide.io/Tools/fsac.html</PackageProjectUrl>
```

Comment:
On my fork I was actually a bit irritated at this - this data (at least the RepositoryUrl) should be available in the build already. I dug into it a bit and realized that the `RepositoryUrl` is set _by the NuGet Pack targets_, but it's only done a) for `Pack` and b) if the user has a few other sentinel properties set up (which usually are handled by the SDK/SourceLink by default). 

So I guess I think that FSharp.Formatting should ideally be able to recognize [this pattern](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.targets#L293) and adhere to it, in the absence of that getting fixed/changed at the SDK level?